### DISCUSSION TITLE
SubGroups of Servers and Sorting

### DISCUSSION THREAD

**Discussion by EliasB-NU** [pelican-dev/panel#561] [Ideas]
Hi, 

I am doing a lot of Minecraft server networks with velocity and so on, I would wish for creating groups with the different servers in the panel, similar to the Hetzner cloud server groups (image below). 
![Screenshot 2024-08-24 150749](https://github.com/user-attachments/assets/b74b0832-4970-4094-8150-e12610c39bc1)
And in addition sorting, in the current instance of pterodactyl I am running servers are sorted by time of creation, if you for example create the proxy at a later point, and after that some other servers it is in the middle of the list, which is in my opinion annoying and not very clean.

So I would wish for some kind of grouping/sorting system in the frontend. 
Please add on your ideas and opinion.

**Comment by trenutoo** [comment]
Already planned, see #249

Duplicates https://github.com/pelican-dev/panel/discussions/26 https://github.com/pelican-dev/panel/discussions/484
